mage_ai/ai/llm_pipeline_wizard.py

```diff
@@ -154,16 +196,56 @@ async def async_generate_block_with_description(self, block_description: str) ->
             function_args = json.loads(response_message["function_call"]["arguments"])
             block_type, block_language, pipeline_type, config = self.__load_template_params(
                 function_args)
-            return fetch_template_source(
+            block_template = fetch_template_source(
                 block_type=block_type,
                 config=config,
                 language=block_language,
                 pipeline_type=pipeline_type,
             )
+            block_uuid = uuid.uuid4().hex
+            return Block(name=block_uuid,
```

Comment:
You don't need to create the block object here. The block object can't be returned as API response. Just return a dictionary of the config.